docs/new-feature-tutorial.md

```diff
@@ -318,11 +324,10 @@ in `zerver/views/home.py`.
         # ...
       )
 
-Since this feature also adds a checkbox to the admin page and adds a
-new property the Realm model that can be modified from there, you
-need to make changes to the `update_realm` function in
-`zerver/views/realm.py`. Add a parameter for the new field to the
-`update_realm` function.
+Since this feature adds a checkbox to the admin page and a new property
+to the Realm model that can be modified from there, you need to add a
+parameter for the new field to the update_realm function in
```

Comment:
Is there a reason you removed the backticks around `update_realm`?

Otherwise looks good!